Directory.Packages.props

```diff
@@ -20,7 +20,7 @@
     <PackageVersion Include="ShortGuid" Version="2.0.1" />
     <PackageVersion Include="ThrottleDebounce" Version="2.0.0" />
     <PackageVersion Include="BlazorMonaco" Version="3.1.0" />
-    <PackageVersion Include="Elsa.Api.Client" Version="3.2.0-preview.1448" />
+    <PackageVersion Include="Elsa.Api.Client" Version="3.2.0-readonly.1556" />
```

Comment:
Let's make sure the core solution is merged first and then update this reference before merging